public/javascripts/ers_scripts.js

```diff
@@ -1,117 +1,128 @@
+        return document.getElementById('input-file-name').files[0].size <= MAX_FILESIZE;
+    }
+}
 
+function removeErrorMsg() {
+    const errorSummary = document.getElementById("error-summary")
+    const checkFileButton = document.getElementById("check-file-button")
+    if (document.querySelector(".govuk-form-group--error")) {
+        document.querySelector(".govuk-form-group--error").classList.remove("govuk-form-group--error")
+    }
 
-	$(document).ready(function () {
+    if (errorSummary) {
+        errorSummary.parentNode.removeChild(errorSummary)
+    }
+    if (checkFileButton) {
+        if (document.getElementById("input-file-name").files.length !== 0) {
+            checkFileButton.disabled = false
+            checkFileButton.focus()
+        } else {
+            checkFileButton.disabled = true
+        }
 
-		/* Spinner */
-		if (ie) {
-			$("#check-file-button").on("click", function (e) {
-				var temp = $("#progress-spinner-img").attr("src");
-				$("#progress-spinner").show();
-				$("#progress-spinner-img").attr("src",temp);
-			});
-		} else {
-			$("#check-file-button").on("click", function (e) {
-				$("#progress-spinner").show();
-			});
-		}
+    }
+}
 
-		$('.validation-summary-message a').on('click', function(e){
-			e.preventDefault();
-				var focusId = $(this).attr('data-focuses'),
-				thingToFocus = $("#"+focusId);
-			$('html, body').animate({
-				scrollTop: thingToFocus.parent().offset().top
-			}, 500);
-			thingToFocus.parent().find('#error-summary').parent().find("input").focus();
-			thingToFocus.parent().find('#fileToUpload').first().focus();
-			thingToFocus.parent().find('.block-label').first().focus();
-			thingToFocus.parent().find('.form-control').first().focus();
-		});
+document.addEventListener("DOMContentLoaded", function () {
+    /* Spinner */
+    if (checkFileButton) {
+        if (ie) {
+            checkFileButton.addEventListener('click', function () {
+                const temp = document.getElementById("progress-spinner-img").getAttribute("src");
+                document.getElementById("progress-spinner").style.display = '';
+                document.getElementById("progress-spinner-img").setAttribute("src", temp)
+            });
+        } else {
+            checkFileButton.addEventListener('click', function () {
+                const spinner = document.getElementById("progress-spinner-img")
+                spinner.style.display = ''
+                spinner.classList.remove("govuk-!-display-none")
+            });
+        }
+    }
 
-		$('#errors').focus();
+    const summaryMessage = document.querySelector('.validation-summary-message a')
+    if (summaryMessage) {
+        document.querySelector('.validation-summary-message a').addEventListener('click', function () {
+            e.preventDefault();
+            const focusId = this.getAttribute('data-focuses'),
+                thingToFocus = document.getElementById(focusId);
+            $('html, body').animate({
```

Comment:
is this jQuery?

Reply:
so it is, good spot! wasn't being used, so removing the whole chunk there